platform/pageModels/hello_page_template.ftl

```diff
@@ -9,6 +9,7 @@
         </title>
         <meta name="viewport" content="width=device-width,  user-scalable=no" />
         <link rel="icon" href="${appUrl}favicon.png" type="image/png" />
+        <link rel="stylesheet" href="<@wp.resourceURL />static/css/rainbow.css" rel="stylesheet">
```

Comment:
I'm not sure this will work. This is referencing a widget static, not a global static. I guess we need a different syntax. Also have a look a this comment: https://entando.myjetbrains.com/youtrack/issue/ENG-4136/Dealing-with-bundle-v5-the-CM-must-be-able-to-manage-resources-not-related-to-any-widget#focus=Comments-4-12845.0-0

Reply:
Thanks. So, I think that it should be something like:

```
bundles/entando-showcase-bundle-c0e268b3/resources/rainbow.css
```

The only problem with this is that if a bundle is initialized using this as template, it is necessary change the bundle id in that reference when the user publishes the derived version.

Reply:
yeah but we should find a way that will work regardless of the bundle id. Maybe something like:
`<link rel="stylesheet" href="javascript:(window.entando.bundleResourcesPath + '/style.css')" />`

Reply:
But we are in a page template here. I could use a page template from a bundle and a widget from a different bundle.

Reply:
For the moment let's put the hardcoded path so we can test it. We'll work on some enhancement later.